### ISSUE TITLE
LineChart XAxis to bottom Legends bottom

### ISSUE THREAD

**Issue by oearias** [PhilJay/MPAndroidChart#3312]
Hi Phill i want to set legends and Xaxis on the bottom chart but the values of both (legends,Xaxis) overlap.
Can i set a space between both?

<img width="312" alt="captura de pantalla 2017-08-04 a la s 14 35 22" src="https://user-images.githubusercontent.com/9981862/28984202-681a31c2-7922-11e7-84f0-15fadb459c0c.png">

**Comment by zugswangcm** [comment]
oraries how u do that . having date as xAxex ?can u teach mu?

**Comment by oearias** [comment]
of course, send me an email and tell me your doubts oscar.enrique.arias@gmail.com

**Comment by zugswangcm** [comment]
Check ur email sir

**Comment by tuandoan23** [comment]
Is there any solustion to this problem? @oearias
